src/ansible/roles/packages/tasks/Fedora.yml

```diff
@@ -1,23 +1,11 @@
-- name: Enable docs installation by default
+- name: Install packages for ground base image
   block:
-  - name: Check dnf.conf exists
-    stat:
-      path: /etc/dnf/dnf.conf
-    register: dnf_stat_result
-
-  - name: Patch dnf.conf
+  - name: Enable docs installation by default
     replace:
       path: /etc/dnf/dnf.conf
       regexp: 'nodocs'
       replace: ''
-    when: dnf_stat_result.stat.exists
 
-- name: Select packageset
```

Comment:
For testing we try to reduce the set of installed packages this is needed for
"Install extended set of packages"